DeveloperGame/DeveloperGame/MultipleChoiceQuestion.cs

```diff
@@ -23,15 +23,33 @@ public override void PrintQuestion()
         {
             base.PrintQuestion();
 
-            foreach (string answer in _answerOptions)
+            for(int i = 0; i < 4; i++)
             {
-                Console.WriteLine(answer);
+                Console.WriteLine((Answers)i + $". {_answerOptions[i]}");
             }
         }
 
         public bool IsTheAnswerCorrect(Answers userAnswer)
         {
             return userAnswer == _correctAnswer;
         }
+
+        public void RandomizeAnswers()
+        {            
+            Random random = new Random();
+
+            string correctAnswer = _answerOptions[(int)_correctAnswer];
+            _answerOptions.Remove(_answerOptions[(int)_correctAnswer]);
+
+            int randomIndex = random.Next(0, 3);
+
+            string temp = _answerOptions[randomIndex];
+
+            _answerOptions[randomIndex] = correctAnswer;
+
+            _answerOptions.Add(temp);
+
+            _correctAnswer = (Answers)randomIndex;
```

Comment:
This is a fairly good algorithm but I have 2 complaints:

1. Leave some comments so it's easier to follow along, I had to draw it out on a napkin to understand what is going on
2. The correct answer can never be D which seems like an issue